src/i18_bluesky/plans/simple_plans.py

```diff
@@ -0,0 +1,88 @@
+from typing import Any, List, Mapping, Optional
+
+import bluesky.plans as bp
+from dls_bluesky_core.core import MsgGenerator
+from ophyd_async.core.detector import StandardDetector
+from ophyd_async.epics.motion import Motor
+
+"""
+Dictionary of scan arguments from motor scan_args (start, stop, steps)
+"""
+
+
+def make_args(motor, scan_args, prefix=""):
+    return {
+        "motor" + prefix: motor,
+        "start" + prefix: scan_args[0],
+        "stop" + prefix: scan_args[1],
+        "steps" + prefix: int(scan_args[2]),
```

Comment:
There's generally 2 ways of defining a plan, either with a fixed step length between points or a number of points. Here you're mixing the two concepts. If you're going to do number of points, stick with `num` rather than `steps` to make it clearer what's going on.

e.g. 

start=0, stop=5, step=2 => {0, 2, 4}
start=0, stop=5, num=2 => {0, 2.5, 5}

(step purposefully chosen to show why I prefer `num`, but users like to think in terms of a fixed step size)